run_mcmc.cc

```diff
@@ -177,6 +177,13 @@ int main(int argc, char *argv[]) {
   bayesmix::read_proto_from_file(args.get<std::string>("--hier-args"),
                                  hier->get_mutable_prior());
 
+  std::cout << "hier->prior: \n"
+            << hier->get_mutable_prior()->DebugString() << std::endl;
+
+  auto updater = std::make_shared<MalaUpdater>(0.25);
+  hier->set_updater(updater);
+  hier->initialize();
+
   // Read data matrices
   Eigen::MatrixXd data =
       bayesmix::read_eigen_matrix(args.get<std::string>("--data-file"));
```

Comment:
Do we need to think a way to pass (maybe via proto) required parameters for updaters like MH or MALA?

Reply:
Yeah I can take care of that!